sale_order_price_recalculation/views/sale_order_view.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_sales_order_auto_done_setting" model="ir.ui.view">
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_sales_order_auto_done_setting" />
```

Comment:
@pedrobaeza Why did you change the original anchor for inheritance? See https://github.com/OCA/sale-workflow/pull/1155/files#diff-5a3cebb009ef27460435af3553b111aaR6
Is there a particular functional reason? Because if you do not activate the option "Lock Confirmed Sales", users won't see the recalculation buttons.

Reply:
Sorry for the late answer. Previous xpath was not working anymore since @legalsylvain's change in  38ffb1389098431d63887a47ff8dbde19042d15e (which both the one you mentioned -  #1155 - and #1245 don't include) and I looked for an alternative without checking the point you mention. I can change it to a more suitable one if this finally goes on.